paasta_tools/cli/utils.py

```diff
@@ -636,34 +634,44 @@ def run_paasta_cluster_boost(
 
 
 def execute_paasta_cluster_boost_on_remote_master(
-    cluster,
+    clusters,
     system_paasta_config,
     action,
-    region,
     pool,
     duration=None,
     override=None,
     boost=None,
     verbose=0,
 ):
     """Returns a string containing an error message if an error occurred.
-    Otherwise returns the output of run_paasta_metastatus().
+    Otherwise returns the output of run_paasta_cluster_boost().
     """
-    try:
-        master = connectable_master(cluster, system_paasta_config)
-    except NoMasterError as e:
-        return (255, str(e))
+    output = {}
+    for cluster in clusters:
+        try:
+            master = connectable_master(cluster, system_paasta_config)
+        except NoMasterError as e:
+            output[cluster] = (255, str(e))
+
+        output[cluster] = run_paasta_cluster_boost(
+            master=master,
+            action=action,
+            pool=pool,
+            duration=duration,
+            override=override,
+            boost=boost,
+            verbose=verbose,
+        )
 
-    return run_paasta_cluster_boost(
-        master=master,
-        action=action,
-        region=region,
-        pool=pool,
-        duration=duration,
-        override=override,
-        boost=boost,
-        verbose=verbose,
-    )
+    aggregated_code = 0
+    aggregated_stdout = ""
+    for cluster in output:
+        code = output[cluster][0]
+        stdout = output[cluster][1]
+        if not code == 0:
+            aggregated_code = code
```

Comment:
You append to aggregated_stdout, but you will only get the aggregated_code of the last failure. This might cause confusion when trying to interpret the stdout. Also, not sure if you also want to save stderr to be safe.

Reply:
If two cluster have different non-zero return code, what code should be returned?

Reply:
@matfra usually we return max()
https://github.com/Yelp/paasta/blob/master/paasta_tools/paasta_serviceinit.py#L248